openmetadata-service/src/main/java/org/openmetadata/service/resources/pipelines/PipelineResource.java

```diff
@@ -170,7 +170,7 @@ public ResultList<Pipeline> list(
   public EntityHistory listVersions(
       @Context UriInfo uriInfo,
       @Context SecurityContext securityContext,
-      @Parameter(description = "pipeline Id", schema = @Schema(type = "string")) @PathParam("id") UUID id)
+      @Parameter(description = "Id of the pipeline", schema = @Schema(type = "UUID")) @PathParam("id") UUID id)
       throws IOException {
```

Comment:
Let's use one convention. Either for all entities "Id of the entity" or "Entity Id"